R/crickets/crickets-map-to-DarwinCore.R

```diff
@@ -0,0 +1,261 @@
+# Map cricket data to Darwin Core ####
+
+# Author: Cherine Jantzen
+# Created: 2024-02-07
+# Last updated: 2024-02-19
+
+
+# I. Preparation ----------------------------------------------------------
+
+# load packages
+library(dplyr)
+library(tidyr)
+library(here)
+
+source(here::here("R", "retrieveData-API-Dataverse.R"))
+
+# Retrieve data from Data station
+
+dataverse_list <- retrieve_dataverse_data(dataset = "doi:10.17026/dans-zsa-f3y9",
+                                          server = "lifesciences.datastations.nl")
+
+
+purrr::map2(.x = c("gryllus", "meta_info_gryllus", "meta_info_plantchem", "plantchem"),
+            .y = dataverse_list[names(dataverse_list) %in% c("gryllus", "meta_info_gryllus", "meta_info_plantchem", "plantchem")],
+            .f = ~{
+              
+              assign(.x, .y, envir = .GlobalEnv)
+              
+            })
+
+
+# II. Mapping of plant data  ----------------------------------------------
+
+## create eventID: event is measure per plot (within one block) 
+plants <- plantchem %>% 
+  dplyr::mutate(eventID = paste0(paste0(substring(Block, 1, 2), substring(Block, 4, 4)), Treat))
```

Comment:
```suggestion
  dplyr::mutate(eventID = paste0(substring(Block, 1, 2), substring(Block, 4, 4), Treat))
```
Wouldn't this result in the same?